scripts/fast-reboot

```diff
@@ -430,6 +436,13 @@ then
     exit "${EXIT_FAILURE}"
 fi
 
+# re-run the script in background mode with detaching from the terminal session
+if [[ x"${DETACH}" == x"yes" ]]; then
+   echo "Detaching the process from the terminal session. Redirecting output to ${LOG_PATH}."
+   $0 "$@" -d 1>$LOG_PATH 2>$1 & disown %%
```

Comment:
2>$1?

Reply:
It means redirect stderr to stdout. stdout in our case is directed to the file.

Reply:
I suspect you wanted to do so, you can use 2>&1 or change the whole thing to 

$0 "$@" -d &>$LOG_PATH & disown %%

Reply:
Done.